internal_transfers/actions/src/accounting.ts

```diff
@@ -64,28 +64,69 @@ export async function simulateSolverSolution(
     return null;
   }
 
-  const commonSimulationParams = {
-    contractAddress: SETTLEMENT_CONTRACT_ADDRESS,
-    sender: solverAddress,
-    value: "0",
-    blockNumber: competition.simulationBlock,
-  };
-  const simFull = await simulator.simulate({
-    ...commonSimulationParams,
-    callData: competition.fullCallData,
-  });
-  const simReduced = await simulator.simulate({
-    ...commonSimulationParams,
-    callData: competition.reducedCallData,
+  const { full, reduced } = await simulateBoth(simulator, {
+    full: competition.fullCallData,
+    reduced: competition.reducedCallData,
+    common: {
+      contractAddress: SETTLEMENT_CONTRACT_ADDRESS,
+      sender: solverAddress,
+      value: "0",
+      blockNumber: competition.simulationBlock,
+    },
   });
+
   return {
     txHash: transaction.hash,
     winningSettlement: competition,
-    full: simFull,
-    reduced: simReduced,
+    full,
+    reduced,
   };
 }
 
+interface commonSimulationParams {
+  contractAddress: string;
+  sender: string;
+  value: string;
+  blockNumber: number;
+}
+
+interface SettlementSimulationParams {
+  full: string;
+  reduced: string;
+  common: commonSimulationParams;
+}
+
+interface SimulationPair {
+  full: SimulationData;
+  reduced: SimulationData;
+}
+async function simulateBoth(
```

Comment:
Nit: Keep consistent with other whitespaces.
```suggestion
}

async function simulateBoth(
```

Reply:
Totally! Im surprised the linter didn't pick this up.